crates/api_common/src/test_utils.rs

```diff
@@ -0,0 +1,33 @@
+#![allow(clippy::unwrap_used)]
```

Comment:
Remove this, its unecessary.

Reply:
Will be addressed as per https://github.com/LemmyNet/lemmy/pull/3742#discussion_r1282137989